### PR TITLE
add lazydocker package

### PR DESCRIPTION
**Pull request by kranurag7**
continuation of an old trend to be little lazy in terminal https://github.com/wolfi-dev/os/pull/11922 :)

### CODE REVIEW

File: lazydocker.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    with:
+      repository: https://github.com/jesseduffield/lazydocker
+      tag: v${{package.version}}
+      expected-commit: be051153525b018a46f71a2b2ed42cde39a1110c
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: |
+        -s -w
```
Comment (anushkamittal20) [inline lazydocker.yaml:20]: Do we need -w? I thought it is set by default 🤔

Comment (kranurag7) [reply lazydocker.yaml:20]: good catch, copy paste mistake from lazygit, correcting it.

Comment (kranurag7) [inline lazydocker.yaml:20]:
```suggestion

```

**Review by anushkamittal20 — APPROVED** [review]
Thanks for the work on this @kranurag7! LGTM

**Pull request merged**